### ISSUE TITLE
Chain index stop syncing after slot 62510369, maybe related with new Babbage era

### ISSUE THREAD

**Issue by manupadillaph** [input-output-hk/plutus-apps#602]
### Summary

Chain index is not syncing in tesnet after Slot: 62510369, BlockId: d931221f9bc4cae34de422d9f4281a2b0344e86aac6b31eb54e2ee90f44a09b9

I converted to POSIX TIME and is: Friday, 22 July 2022 8:19:45.999 AM, related with the time frame of the new babbage era.

I download and built the latest plutus apps (commit 139c0429f2c7a656ff6114e6373f895d29132860) and syncing from zero again with no success. Im using with cardano-node-1.35.0-linux which is syncing 100%.

There was similar issues with cardano-db-sync and they said they fix the issue there. https://github.com/input-output-hk/cardano-db-sync/issues/1181 https://github.com/input-output-hk/cardano-db-sync/issues/1182

Here also, some report about the chain index rolling back slots after this slot: https://github.com/input-output-hk/plutus-apps/pull/557

Any idea?

Thank you

### Steps to reproduce the behavior

Install Cardano Node 1.35
Build Latest Plutus-Apps
Run Chain Index

### Actual Result

Stop syncint and stayt with this message:

[chain-index:Info:33] [2022-07-16 23:01:46.14 UTC] Syncing (98.22%). Processed 0 blocks, 1 rollbacks in the last 13s. Current tip is Point(Slot 62510369, BlockId d931221f9bc4cae34de422d9f4281a2b0344e86aac6b31eb54e2ee90f44a09b9)

### Expected Result

Continue Syncing until 100%

### Describe the approach you would take to fix this

Something similar to what has been made in cardano-db-sync?

I fount they used eras/babbage/impl in cardano-ledger in the latest cabal file.
I culdnét make this work inside the cabal file of plutus app

source-repository-package
  type: git
  location: https://github.com/input-output-hk/cardano-ledger
  tag: 3be8a19083fc13d9261b1640e27dd389b51bb08e
  --sha256: 0dvm9l43mp1i34bcywmznd0660hhcfxwgawypk9q1hjkml1i41z3
  subdir:
    eras/alonzo/impl
    -- eras/alonzo/test-suite
    eras/babbage/impl

### System info

WSL2
Ubuntu

**Comment by manupadillaph** [comment]
There is any schedule for the new release of plutus app and chain index working with baggage era?

**Comment by ak3n** [comment]
Yes, please check the `next-node` branch and don't forget to set a different `chain-index.db` path.

**Comment by manupadillaph** [comment]
great, Im trying now. So, it need to sync in a  new database?

**Comment by koslambrou** [comment]
I would suggest using the `cardano-node` builded with the Nix shell of `plutus-apps`. Currently, it's using the `1.35.2` version.

**Comment by manupadillaph** [comment]
Ok. Thank you so much. 
Im trying building the node with nix as indicated here: https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/building-the-node-using-nix.md/ 
What do you mean with Nix shell of plutus-apps?
Im doing the build inside the nix shell

**Comment by koslambrou** [comment]
That's work as well. But `cardano-node` is already built with `plutus-apps`. Following the instructions in the README of plutus-apps. Install Nix, setup the Nix cache, then do `nix-shell`. From there, you'll have access to the `cardano-node` executable.
